rllib/utils/actor_manager.py

```diff
@@ -599,6 +605,7 @@ def foreach_actor(
     def foreach_actor_async(
         self,
         func: Union[Callable[[Any], Any], List[Callable[[Any], Any]]],
+        tag: str = "default",
```

Comment:
so thinking about this a bit more, it would be the best if we can attach multiple tags to an async all, like basically this parameter should be tags:
```
for sync alls, tags would be ()
for async call users, they can attached multiple tags for a single call, like ("rollout_worker", "sync_weight")
for async fetch result, we can also specify a single, or list of tags. for example, ("eval", "sample") will fetch all the sample() calls on eval workers.
```
I understand this is back & forth a few times, not sure if you are interested in making the changes.
if not, we can merge and unblock. you for now, and I can make these updates later today.

Reply:
actually can I add a todo here and we punt on this?

Reply:
We need this pr to implement the async trainer runner and I am falling behind atm

Reply:
I've added a todo as a comment for now